solr/datadog_checks/solr/data/conf.yaml.example

```diff
@@ -1,38 +1,113 @@
 instances:
+
+  ## @param host - string - required
+  ## Solr host to connect to.
+  #
   - host: localhost
+
+  ## @param port - integer - required
+  ## Solr port to connect to.
+  #
     port: 9999
-  #   user: username
-  #   password: password
-  #   process_name_regex: .*process_name.* # Instead of specifying a host, and port. The agent can connect using the attach api.
-  #                                                                       #   This requires the JDK to be installed and the path to tools.jar to be set below.
-  #   tools_jar_path: /usr/lib/jvm/java-7-openjdk-amd64/lib/tools.jar # To be set when process_name_regex is set
+
+  ## @param user - string - optional
+  ## Username from the credentials needed to connect to the host.
+  #
+  #  user: <USERNAME>
+
+  ## @param password - string - optional
+  ## Password from the credentials needed to connect to the host.
+  #
+  #  password: <PASSWORD>
+
+  ## @param process_name_regex - string - optional
+  ## Instead of specifying a host, and port. The agent can connect using the attach api.
+  ## This requires the JDK to be installed and the path to `tools.jar` to be set below in `tools_jar_path` parameter.
+  #
+  #  process_name_regex: .*process_name.*
+
+  ## @param tools_jar_path - string - optional
+  ## Needs to be set when process_name_regex parameter is set.
+  #
+  #  tools_jar_path: /usr/lib/jvm/java-7-openjdk-amd64/lib/tools.jar
+
+  ## @param name - string - optional
+  ## XXXXXXXXXXX
```

Comment:
What could be the definition for the name parameter?

Reply:
it is the instance name yes. Used here https://github.com/DataDog/jmxfetch/blob/c86d6b3d2ac951856027d2e5a6188cdeac42dc40/src/main/java/org/datadog/jmxfetch/Instance.java#L154-L171

Reply:
This parameter exists for every imx integration actually

Reply:
Do you know how this is used?

Reply:
Shouldn't we move it to the top then?

Reply:
It is used for tagging purposes

Reply:
your call, i'm fine with both of them since this parameter is optional